rust/gui-client/src-common/src/deep_link/linux.rs

```diff
@@ -137,12 +137,18 @@ Categories=Network;
 
     // Needed for Ubuntu 22.04, see issue #4880
     let update_desktop_database = "update-desktop-database";
-    let status = Command::new(update_desktop_database)
-        .arg(&dir)
-        .status()
-        .with_context(|| format!("failed to run `{update_desktop_database}`"))?;
-    if !status.success() {
-        bail!("{update_desktop_database} returned failure exit code");
+    match Command::new(update_desktop_database).arg(&dir).status() {
+        Ok(status) => {
+            if !status.success() {
+                bail!("{update_desktop_database} returned failure exit code");
+            }
+        }
+        Err(e) if e.kind() == ErrorKind::NotFound => {
+            // This is not an Ubuntu machine, so this executable won't exist.
```

Comment:
Can we log a `debug` message here please that the command as not been found?

Reply:
Of course, I've update the PR